app/serializers/oembed_serializer.rb

```diff
@@ -40,8 +40,7 @@ def html
     attributes = {
       src: embed_short_account_status_url(object.account, object),
       class: 'mastodon-embed',
-      frameborder: '0',
-      scrolling: 'no',
+      style: 'width: 400px; max-width: 100%; border: none;',
       width: width,
```

Comment:
Style should use the width from the variable

Reply:
I misunderstood width setting cannot be overwritten by max-width in style settings. I fixed it. Thanks!